Public/Src/Engine/Scheduler/Distribution/Worker.cs

```diff
@@ -262,10 +262,12 @@ public virtual void Start()
         /// <summary>
         /// Signals that build is finished and that worker should exit
         /// </summary>
-        public virtual void Finish(string buildFailure)
+#pragma warning disable 1998

```

Comment:
Can we use SuppressMessage here instead? If not, can you add a comment describing the meaning of '1998'?

Reply:
suppressmessage does not work here. I added a comment though